packages/react-components/src/Popup/PopupWindow.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017-2021 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { PopupWindowProps, WindowProps } from './types';
+
+import React from 'react';
+import { createPortal } from 'react-dom';
+import styled, { css } from 'styled-components';
+
+import { useElementPosition } from '@polkadot/react-hooks';
+
+import { getPosition } from './utils';
+
+export const POINTER_OFFSET = 14 * 0.8;
```

Comment:
Could you put a line of docs - why such values?

Reply:
✅